tsconfig.json

```diff
@@ -17,134 +17,137 @@
 		"./typings/tsd.d.ts"
 	],
 	"files": [
-		"./src/DateObject.ts",
```

Comment:
My point is that if I run `grunt dev` inside the repository, I get a bunch of changes right back to the order this file already was in.  Shouldn't `grunt dev` typically be getting run before commits (without atom-typescript running to immediately clobber it again, I thought we had generally stopped using that)?